server/cmd/upload.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+
+	"github.com/odpf/stencil/server/api/v1/pb"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+)
+
+var namespace, name, version, filePath string
+var latest bool
+
+func init() {
+	uploadCmd := &cobra.Command{
+		Use:   "upload",
+		Short: "Upload filedescriptorset file",
+		Run:   upload,
+		Args:  cobra.NoArgs,
+	}
+	uploadCmd.Flags().StringVarP(&namespace, "namespace", "g", "", "provide namespace/group or entity name")
+	uploadCmd.MarkFlagRequired("group")
+	uploadCmd.Flags().StringVarP(&name, "name", "n", "", "provide proto repo name")
+	uploadCmd.MarkFlagRequired("name")
+	uploadCmd.Flags().StringVarP(&version, "version", "v", "", "provide semantic version compatible value")
+	uploadCmd.MarkFlagRequired("version")
+	uploadCmd.Flags().BoolVarP(&latest, "latest", "l", false, "mark as latest version")
+	uploadCmd.Flags().StringVarP(&filePath, "file", "f", "", "provide path to fully contained file descriptor set file")
+	uploadCmd.MarkFlagRequired("file")
+	rootCmd.AddCommand(uploadCmd)
+
+}
+
+func upload(cmd *cobra.Command, args []string) {
+	fileData, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		log.Fatalln("Unable to read provided file", err)
+	}
+	conn, err := grpc.Dial("localhost:8000", grpc.WithInsecure())
```

Comment:
Why is this host hardcoded?

Reply:
Ya this should be removed. thanks. I was testing it out initially. For adding cli commands will rise separate PR.